### PR TITLE
Feature/sc 534

### PR DESCRIPTION
**Pull request by DanCh11**

### CODE REVIEW

**Review by costezki — APPROVED** [review]
does this work now?

**Review by costezki — CHANGES_REQUESTED** [review]
I see tests failing. Can you please check those?

**Review by Dragos0000 — APPROVED** [review]

**Review by Dragos0000 — APPROVED** [review]

**Review by Dragos0000 — APPROVED** [review]

**Review by Dragos0000 — APPROVED** [review]

File: tests/unit/dags/test_crawl_pipeline.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 fake_filename = 'fake_file.json'
 fake_bucket_name = 'fake-bucket-name'
 fake_es_index_name = 'fake_index_name'
-fake_content_path_key = 'fake_content'
+fake_content_path_key = 'title'
```
Comment (costezki) [inline tests/unit/dags/test_crawl_pipeline.py:16]: it is strange to see the title passed as content.

**Review by costezki — APPROVED** [review]
are we sure we want to pass title for content?

**Pull request merged**